shared-module/displayio/I2CDisplay.c

```diff
@@ -74,15 +73,31 @@ void common_hal_displayio_i2cdisplay_deinit(displayio_i2cdisplay_obj_t* self) {
     reset_pin_number(self->reset.pin->number);
 }
 
-bool common_hal_displayio_i2cdisplay_begin_transaction(mp_obj_t obj) {
+bool common_hal_displayio_i2cdisplay_reset(mp_obj_t obj) {
+    displayio_i2cdisplay_obj_t* self = MP_OBJ_TO_PTR(obj);
+    if (self->reset.base.type == &mp_type_NoneType) {
+        return false;
+    }
+
+    common_hal_digitalio_digitalinout_set_value(&self->reset, false);
+    common_hal_mcu_delay_us(1);
```

Comment:
SSD1306 datasheet says min 3us. SSD1331 (color) is 3us, SSD 1351 is min 2us. I don't know of other I2C bitmap displays.
SSD1306:
![image](https://user-images.githubusercontent.com/2847802/63594516-a10f4c00-c584-11e9-9521-508b50b76a1c.png)